Glossary: argument to ascii (#28718)
mdn/content: files/en-us/glossary/ascii/index.md

{{GlossarySidebar}}

> **ASCII** (_American Standard Code for Information Interchange_) is a character encoding standard of 128 7-bit used by computers for converting letters, numbers, punctuation and control codes into digital form.

Reviewer: ```suggestion
**ASCII** (_American Standard Code for Information Interchange_) is a character encoding standard of 128 7-bit used by computers for converting letters, numbers, punctuation, and control codes into digital form.
```

> **ASCII** (_American Standard Code for Information Interchange_) is a character encoding standard of 128 7-bit used by computers for converting letters, numbers, punctuation and control codes into digital form.

Reviewer 2: ```suggestion
**ASCII** (_American Standard Code for Information Interchange_) is a 128-code point, 7-bit character encoding standard used by computers for converting letters, numbers, punctuation, and control codes into digital form.
```

> ASCII has 128 code points. The first 33 points are non-printing control codes including the carriage return, line feed, tab, and several obsolete non-printable code stemming from its origin as telegraph codes. The other 95 are printable characters, including digits (0-9), lowercase (a-z) and uppercase (A-Z) letters, and punctuation symbols.

Reviewer 2: ```suggestion
The first 33 ASCII code points are non-printing control codes including the carriage return, line feed, tab, and several obsolete non-printable codes stemming from its origin of representing telegraph codes. The other 95 are printable characters, including digits (0-9), lowercase (a-z) and uppercase (A-Z) letters, and punctuation symbols.
```

> Many computer systems instead use {{glossary("Unicode")}}, which has millions of code points, but the first 128 of these are the same as the ASCII set. Since 2007, {{Glossary("UTF-8")}} superseded it on the Web.

Reviewer 2: ```suggestion
Many computer systems instead use {{glossary("Unicode")}}, which has millions of code points, but the first 128 of these are the same as the ASCII set. {{Glossary("UTF-8")}} superseded ASCII on the Web in 2007.
```
